### ISSUE TITLE
Akka 2.6.20 might have broken the usage of Aeron UDP

### ISSUE THREAD

**Issue by Zhen-hao** [akka/akka#31576]
After upgrading to Akka 2.6.20, I saw extremely high CPU usage and unrecoverable Aeron runtime error on a cluster running Java17. 
I hope the following logs can help diagnose the issue. 

```
Sep 10 17:42:16 app1 nt-ui[279640]: Fatal Aeron error DriverTimeoutException. Have to terminate ActorSystem because it lost contact with the embedded Aeron media driver. Possible configuration properties to mitigate the problem are 'client-liveness-timeout' or 'driver-timeout'. io.aeron.exceptions.DriverTimeoutException: FATAL - MediaDriver keepalive (ms): age=20992 > timeout=20000
Sep 10 17:42:16 app1 nt-ui[279640]: Cluster Node [akka://nt-ui@172.16.16.122:2552] - Scheduled sending of heartbeat was delayed. Previous heartbeat was sent [5486] ms ago, expected interval is [1000] ms. This may cause failure detection to mark members as unreachable. The reason can be thread starvation, CPU overload, or GC.
Sep 10 17:42:16 app1 nt-ui[279640]: Cluster Node [akka://nt-ui@172.16.16.122:2552] - Scheduled sending of heartbeat was delayed. Previous heartbeat was sent [236324] ms ago, expected interval is [1000] ms. This may cause failure detection to mark members as unreachable. The reason can be thread starvation, CPU overload, or GC.
Sep 10 17:42:16 app1 nt-ui[279640]: Aeron error, org.agrona.concurrent.AgentTerminationException
```

and 

```
Sep 11 03:33:12 app1 nt-ui[281262]: Fatal Aeron error DriverTimeoutException. Have to terminate ActorSystem because it lost contact with the embedded Aeron media driver. Possible configuration properties to mitigate the problem are 'client-liveness-timeout' or 'driver-timeout'. io.aeron.exceptions.DriverTimeoutException: FATAL - MediaDriver keepalive (ms): age=20341 > timeout=20000
Sep 11 03:33:13 app1 nt-ui[281262]: Running CoordinatedShutdown with reason [ActorSystemTerminateReason]
Sep 11 03:33:14 app1 nt-ui[281262]: Aeron error, org.agrona.concurrent.AgentTerminationException
``` 

After reverting back to Akka 2.6.19, the Aeron error is gone. 
I can confirm the Aeron version isn't a problem. With Akka 2.6.19, both Aeron 1.39.0 and 1.38.2 work without giving the same runtime error as with Akka 2.6.20.

**Comment by patriknw** [comment]
Thanks for reporting. We will look into what could be the reason for that.

**Comment by Zhen-hao** [comment]
I was wrong.

The problem is actually with the new Aeron version.

Although with Akka 2.6.19 and Aeron 1.39.0 the above error is gone, there are silent failures.
All I can see is that every node can not connect to any seed nodes.

For example,
```
Sep 12 20:41:53 app3 nt-ui[305104]: Cluster Node [akka://nt-ui@172.16.18.110:2552] - Couldn't join seed nodes after [211] attempts, will try again. seed-nodes=[akka://nt-ui@172.16.17.54:2552]
```

After changing Aeron back to 1.37.0, the problem is gone.

**Comment by He-Pin** [comment]
> I was wrong.
> 
> The problem is actually with the new Aeron version.
> 
> Although with Akka 2.6.19 and Aeron 1.39.0 the above error is gone, there are silent failures.
> All I can see is that every node can not connect to any seed nodes.
> 
> For example,
> ```
> Sep 12 20:41:53 app3 nt-ui[305104]: Cluster Node [akka://nt-ui@172.16.18.110:2552] - Couldn't join seed nodes after [211] attempts, will try again. seed-nodes=[akka://nt-ui@172.16.17.54:2552]
> ```
> 
> After changing Aeron back to 1.37.0, the problem is gone. 
> 
> 

So akka 2.6.20 works with the older aeron right？

**Comment by Zhen-hao** [comment]
> So akka 2.6.20 works with the older aeron right？

I didn't test that. I'm now running 2.6.19.

**Comment by Zhen-hao** [comment]
The real problem might just be a particular network issue on my vms on AWS . 
The New Aeron version might not be the cause of the problem, because I'm seeing the `Couldn't join seed nodes after [988] attempts, will try again.` error from one node again. (I will replace the problematic node with a new vm.)

It is just the behavior of Akka 2.6.20 is different from 2.6.19 when facing this issue.

**Comment by Zhen-hao** [comment]
A new VM indeed solved the network issue...
I don't know what caused the network issue in the first place. It is unlikely but possible to be caused by deploying Akka 2.6.20. 

Feel free to close this issue because I don't know how to reproduce it publicly.

**Comment by patriknw** [comment]
Thanks for digging further.

**Comment by Zhen-hao** [comment]
The root cause of my reported problem with the Akka `2.6.20` is a missing dependency.

I really need both 
```scala
 "org.agrona"     % "agrona"    % AgronaVersion pomOnly (),
 "uk.co.real-logic"    % "sbe-tool"    % SbeToolVersion pomOnly (),
```
in my `libraryDependencies`.

I deleted them when I upgraded the Akka and Aeron versions, forgetting [the reasons](https://github.com/akka/akka/issues/29982) why I had added them in the first place.

With those two lines added back, Akka `2.6.20` works just fine.
